data/language/en-GB.txt

```diff
@@ -4309,6 +4309,15 @@ STR_5997    :Add/set money
 STR_5998    :Add money
 STR_5999    :Set money
 STR_6000    :Enter new value
+STR_6001    :Cut-away View
+STR_6002    :Cut-away View
```

Comment:
One is the menu entry, the other is the window title.
The same text works for both in english.
I thought it best to keep them separate in case this will not be the case in some localisations.